src/TLuaInterpreter.cpp

```diff
@@ -248,58 +248,105 @@ int TLuaInterpreter::Wait(lua_State* L)
     return 0;
 }
 
-// Documentation: ? - public function missing documentation in wiki
-QString TLuaInterpreter::dirToString(lua_State* L, int position)
+// Documentation: PRIVATE function with no documentation in wiki, converts an
+// exit direction number (in range 1 to 12) to corresponding text string used
+// as the key from them in room exit weight and custom exit line members of the
+// TRoom class. Note that a previous incarnation of this function returned the
+// complete lower case words in all cases which would break should those numbers
+// have been used - at present the Wiki does NOT mention the numbers which is
+// deliberate IIRC to prevent that borkage from showing... SlySven Nov 2018
+QString TLuaInterpreter::dirToString(lua_State* L, int position, bool isForCustomExits)
 {
     QString dir;
     int dirNum;
     if (lua_isnumber(L, position)) {
         dirNum = lua_tonumber(L, position);
-        if (dirNum <= 0 || dirNum >= 13) {
+        switch (dirNum) {
+        // breaks not needed - all handled cases end in a return!
+        case 1:
+            if (isForCustomExits) {
```

Comment:
There is no *after the switch* for the handled cases in the current proposal because I use `return` to immediately leave the function - it is not the most elegant solution but it is a fast one...